### PR TITLE
Adding End User Working Group structure & content

### PR DESCRIPTION
**Pull request by sharrmander**
Adding a new folder to store end user working group assets. 

New assets included in this PR include a working group README and the new slack discussion group code of conduct. 

In the future I'm planning to add a survey directory with links and results to promote transparency and easy discoverability and other content as it is developed.

### CODE REVIEW

**Review by arminru — COMMENTED** [review]
Great to see this and future content published in a proper repo! Having the content spread over numerous Google Docs files owned by individuals or in Slack threads feels much less organized and rather ephemeral. Thank you for that! 🙂

File: enduser-WG/discussion-group-code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# #OTel-EndUsers Code of Conduct
```
[the remark this answers is not in the document]
Comment (sharrmander) [reply enduser-WG/discussion-group-code-of-conduct.md]: I'm accustomed to the naming structure '#channel-name' to indicate a slack channel; I wonder if it's just me?

**Review by arminru — APPROVED** [review]

**Review by mtwo — APPROVED** [review]

**Review by alolita — APPROVED** [review]
lgtm.

File: enduser-WG/discussion-group-code-of-conduct.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (arminru) [reply enduser-WG/discussion-group-code-of-conduct.md]: Ah I see. That one would be `#otel-user-research` though, right?

**Pull request merged**